### ISSUE TITLE
Support importing of multiple collections at once and/or postman full backup.

### ISSUE THREAD

**Issue by Tha14** [flawiddsouza/Restfox#25]
Currently, it's very tedious to import a large number of collections from Postman in order to migrate. Allowing multi-select in the file selection screen and then running the import for each file would be very nice to have.

Also, Postman allows you to export all of your data in a complete dump file(in json format). Within that backup the collections are present in an array so it doesn't seem hard to parse(possibly the same way it is currently done in Restfox). It might also be a nice to have feature for people migrating.

**Comment by flawiddsouza** [comment]
@Tha14  You can now import multiple files at once. Just implemented it: https://github.com/flawiddsouza/Restfox/commit/087ab4f0d01ab6d9e5b0cbaa5987138a9823f849. I'll try to work on supporting the postman dump meanwhile.

**Comment by Sam-Hall** [comment]
After the Postman update that wiped out scratchpad, I was able to import all my collections to Restfox from a json backup found in %appdata%\Postman using the node script in this Stack Overflow post... https://stackoverflow.com/questions/77142871/how-to-convert-postman-backup-file-into-collection-files
